### DISCUSSION TITLE
Feature: Tkinter GUI for adding word data to SpaCy

### DISCUSSION THREAD

**Discussion by ghost** [explosion/spaCy#2243] [New Features & Project Ideas]
It would be very convenient if there was a Tkinter window with fields for adding word data. For example, the word 'cook':

```
----------------------------------------------
|  [ cook ]         ✅ Already recognized    |
| PARTS OF SPEECH                             |
| - Verb                     ➕ Add new POS  |
----------------------------------------------
```

etc.

This would be very useful for human-sourced information (i.e. language information not currently available on the web).

**Comment by honnibal** [comment]
I think Tkinter usually doesn't give such a nice user experience. Instead it's much easier to build good user interfaces in the browser. This also gives a natural client/server architecture, so that you can host the app.

We have an annotation tool focussed on text annotations, https://prodi.gy , which probably wouldn't work so well for this type of database entry. I would actually suggest you try something like Airtable: it should be possible to build a reasonably efficient interface, with very little time.
